vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/HypervisorHostHelper.java

```diff
@@ -839,10 +849,79 @@ private static void createPortGroup(String physicalNetwork, String networkName,
         }
     }
 
-    public static boolean isSpecMatch(DVPortgroupConfigInfo currentDvPortgroupInfo, DVPortgroupConfigSpec newDvPortGroupSpec) {
+    private static boolean eitherObjectNull(Object obj1, Object obj2) {
+        return (obj1 == null && obj2 != null) || (obj1 != null && obj2 == null);
+    }
+
+    private static boolean areBoolPoliciesDifferent(BoolPolicy currentPolicy, BoolPolicy newPolicy) {
+        return eitherObjectNull(currentPolicy, newPolicy) ||
```

Comment:
any case where both objs are null?

Reply:
@sureshanaparti if both objects are null that would mean both policies are essentially same (`null`). Method should return false in that case